crates/extensions/tedge_log_manager/src/actor.rs

```diff
@@ -285,9 +286,7 @@ fn request_from_message(
     if message.payload_bytes().is_empty() {
         Ok(None)
     } else {
-        Ok(Some(LogUploadCmdPayload::from_json(
-            message.payload_str()?,
-        )?))
+        Ok(Some(LogUploadCmd::payload_from(message.payload_bytes())?))
```

Comment:
Beware that this usage of `payload_from` will be made useless by https://github.com/thin-edge/thin-edge.io/pull/2844 which purpose is to use the same pattern for Log&Config management as for Software management: i.e. to have the log manager MQTT independent. The `request_from_message` method will be removed.